### PR TITLE
pypy2: Update to 7.1.1

### PR DESCRIPTION
**Pull request by TheCjw**
Update to latest version and autoupdate is fixed.

### CODE REVIEW

File: bucket/pypy2.json [hunk 1]
```diff
@@ -17,13 +17,10 @@
...
+        "re": "pypy2.7-v([\\d.]+)-win32"
     },
     "autoupdate": {
-        "url": "https://bitbucket.org/pypy/pypy/downloads/pypy2-v$version-win32.zip",
-        "extract_dir": "pypy2-v$version-win32",
-        "hash": {
-            "url": "https://pypy.org/download.html"
-        }
+        "url": "https://bitbucket.org/pypy/pypy/downloads/pypy2.7-v$version-win32.zip",
+        "extract_dir": "pypy2.7-v$version-win32"
```
Comment (Ash258) [inline bucket/pypy2.json]: Add hash extraction as in other version

File: bucket/pypy2.json [hunk 2]
```diff
@@ -1,10 +1,10 @@
 {
-    "version": "6.0.0",
+    "version": "7.1.1",
     "license": "MIT",
```
Comment (Ash258) [inline bucket/pypy2.json]: No description

File: bucket/pypy2.json [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
-    "extract_dir": "pypy2-v6.0.0-win32",
-    "url": "https://bitbucket.org/pypy/pypy/downloads/pypy2-v6.0.0-win32.zip",
+    "extract_dir": "pypy2.7-v7.1.1-win32",
+    "url": "https://bitbucket.org/pypy/pypy/downloads/pypy2.7-v7.1.1-win32.zip",
     "homepage": "https://pypy.org",
-    "hash": "6e2210dae1ae721ed4eb9cba19f15453514b64111511c84f24843c4fdefdaf7f",
+    "hash": "9c59226311f216a181e70ee7b5aa4d9665a15d00f24ae02acec9af7d96355f63",
     "bin": [
```
Comment (Ash258) [inline bucket/pypy2.json]: Same as in other PR

**Comment by TheCjw** [comment]
Fixed and tested.

**Pull request merged**